### PR TITLE
Feature/reuse sites report

### PR DESCRIPTION
**Pull request by SumeN**
For PBI 73324

### CODE REVIEW

File: src/EA.Weee.RequestHandlers/Admin/AatfReports/GetAllAatfReuseSitesCsvHandler.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    using EA.Weee.Requests.Admin.AatfReports;
+    using Prsd.Core.Mediator;
+    using Security;
+    public class GetAllAatfReuseSitesCsvHandler : IRequestHandler<GetAllAatfReuseSitesCsv, CSVFileData>
+    {
+        private readonly IWeeeAuthorization authorization;
+        private readonly WeeeContext weeContext;
+        private readonly CsvWriterFactory csvWriterFactory;
+        private readonly ICommonDataAccess commonDataAccess;
+        public GetAllAatfReuseSitesCsvHandler(IWeeeAuthorization authorization, WeeeContext weeContext,
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.RequestHandlers/Admin/AatfReports/GetAllAatfReuseSitesCsvHandler.cs]: Add some spacing here

File: src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAllAatfReuseSitesCsvDataTests.cs [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    using WeeeReusedSite = Domain.AatfReturn.WeeeReusedSite;
+
+    public class GetAllAatfReuseSitesCsvDataTests
+    {
+        [Fact]
+        public async Task Execute_GivenNoData_NoResultsShouldBeReturned()
+        {
+            using (var db = new DatabaseWrapper())
+            {
+                var @return = ObligatedWeeeIntegrationCommon.CreateReturn(null, db.Model.AspNetUsers.First().Id, FacilityType.Aatf);
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAllAatfReuseSitesCsvDataTests.cs]: Can a test be added to also check the that field values are expected as well as the count... i.e. that the AatfReuseSitesData is populated as expected

File: src/EA.Weee.RequestHandlers/Admin/AatfReports/GetAllAatfReuseSitesCsvHandler.cs (hunk 1 above)
Comment (SumeN) [reply src/EA.Weee.RequestHandlers/Admin/AatfReports/GetAllAatfReuseSitesCsvHandler.cs]: Added

File: src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAllAatfReuseSitesCsvDataTests.cs (hunk 2 above)
Comment (SumeN) [reply src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAllAatfReuseSitesCsvDataTests.cs]: Updated the test

**Pull request merged**